misc/packaging/ansible-runner-service-nginx.spec

```diff
@@ -4,8 +4,8 @@
 %endif
```

Comment:
Can you move this file to "misc/packaging/nginx"? Also it would be needed to rename it as "ansible-runner-service-nginx.spec"

Reply:
Done

Reply:
Thx!